.ycm_extra_conf.py

```diff
@@ -0,0 +1,206 @@
+# This file is NOT licensed under the GPLv3, which is the license for the rest
+# of YouCompleteMe.
```

Comment:
See YouCompleteMe

Reply:
😄 what?

Reply:
Exactly - what!!

The text says "This file is NOT licensed under the GPLv3, which is the license for the rest of YouCompleteMe."

That looks like a placeholder to me.

And it should be obvious I am not actually reviewing this - I just wanted to see what the code looked like :-)

Reply:
I can remove it and say it is licensed as BSD 3-clause like the rest of the repo.

Reply:
Thanks.

Reply:
Actually this file says it is public domain, so I'm not sure what I should do.

Reply:
I thought this was our file. If YouComleteMe is an actual real name of a product called YouCompleteMe then we have to leave that alone - it isn't ours to modify.

Reply:
Yep this file was copied from the YouCompleteMe project, we should leave as is